core/input/devices.js

```diff
@@ -136,10 +136,6 @@
 
             //Util.Debug(">> Keyboard.ungrab");
         },
-
-        sync: function (e) {
-            this._handler.syncModifiers(e);
-        }
```

Comment:
We already release keys when we lose focus, so I'm not sure what scenario that would be needed?